scripts/image_inputs.py

```diff
@@ -22,6 +22,20 @@
 # binary = cv2.imread(join(BINARY,blockO),0)
 binary = cv2.imread(join(BINARY,aims),0)
 
+#RGB images:
+green_cross = 'green_cross.tif' # 3x3 image
+blockO_rgb = 'block_o_RGB.tif' # 10x10 image
+
+BGR = cv2.imread(join(R_G_B,green_cross))
+RGB = cv2.cvtColor(BGR,cv2.COLOR_BGR2RGB)
+
+# BGR = cv2.imread(join(R_G_B,blockO_rgb))
+# RGB = cv2.cvtColor(BGR,cv2.COLOR_BGR2RGB)
```

Comment:
Only keep a single line that's being run. Don't duplicate.
This is convenient when doing your own debugging/testing, but shouldn't be kept in final version.

Remove this and other duplicated & commented out sections (ie. the above blockO_rgb line as well).